perllib/FixMyStreet/SendReport/Open311.pm

```diff
@@ -127,7 +127,7 @@ sub send {
             $revert = 1;
         }
 
-        if ($row->cobrand eq 'fixmybarangay') {
+        if ($row->cobrand eq 'fixmybarangay' || $row->cobrand eq 'greenwich') {
             # FixMyBarangay endpoints expect external_id as an attribute, as do Oxfordshire
```

Comment:
Presumably this comment should say Greenwhich, not Oxfordshire? (Or Oxfordshire should be in the if statement too?)